sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/AutoConfiguredOpenTelemetrySdkBuilder.java

```diff
@@ -331,6 +336,15 @@ public AutoConfiguredOpenTelemetrySdk build() {
 
     ConfigProperties config = getConfig();
 
+    AutoConfiguredOpenTelemetrySdk fromFileConfiguration = maybeConfigureFromFile(config);
+    if (fromFileConfiguration != null) {
+      if (registerShutdownHook) {
+        Runtime.getRuntime()
+            .addShutdownHook(shutdownHook(fromFileConfiguration.getOpenTelemetrySdk()));
+      }
+      return fromFileConfiguration;
```

Comment:
Do we want to support `setResultAsGlobal` as well? (We'll need that for the javaagent usecase)

Reply:
Good point.